openSearchConnection.go

```diff
@@ -308,10 +308,11 @@ func (osc *openSearchConnection) flush(client apiClient, final bool) {
 		} else {
 			osc.mu.Lock()
 			osc.messagesSent += len(logBuffer)
-			osc.mu.Unlock()
 			backoffDuration = 0
```

Comment:
backoffDuration is a local variable in this go routine and doesn't need to be moved into osc critical section.